pkg/apis/build/v1alpha1/build_pod.go

```diff
@@ -116,70 +119,39 @@ func (b *Build) BuildPod(config BuildPodConfig, secrets []corev1.Secret, builder
 					Resources:       b.Spec.Resources,
 				},
 			},
+			SecurityContext: &corev1.PodSecurityContext{
+				FSGroup: &userAndGroup.Gid,
+			},
 			InitContainers: []corev1.Container{
-				{
-					Name:            "creds-init",
-					Image:           config.CredsInitImage,
-					Args:            secretArgs,
-					ImagePullPolicy: corev1.PullIfNotPresent,
-					VolumeMounts:    append(secretVolumeMounts, homeVolume),
-					Env: []corev1.EnvVar{
-						{
-							Name:  "HOME",
-							Value: "/builder/home",
-						},
-					},
-				},
-				{
-					Name:  "source-init",
-					Image: config.SourceInitImage,
-					SecurityContext: &corev1.SecurityContext{
-						RunAsUser:  &root,
-						RunAsGroup: &root,
-					},
-					Env:             b.BuildEnvVars(),
-					ImagePullPolicy: corev1.PullIfNotPresent,
-					WorkingDir:      "/workspace",
-					VolumeMounts: []corev1.VolumeMount{
-						imagePullSecretsVolume,
-						sourceVolume,
-						homeVolume,
-					},
-				},
 				{
 					Name:  "prepare",
```

Comment:
Should we rename this step to `build-init` so that it matches up with the binary name?